### ISSUE TITLE
Strike with no mute and 0 punishment

### ISSUE THREAD

**Issue by cagrimertbakirci** [jagrosh/Vortex#51]
It would be awesome if we could enter 0 as the strike number to just warn the user (or maybe a new command like "warn" would be great).

Thanks a lot!

**Comment by jagrosh** [comment]
If the purpose of this is to keep track of the warning, then it sounds like you should just give the user a strike, and maybe change the punishments list such that the first few strikes don't have punishments (and are effectively just warnings until they get more than a certain number).

If the purpose of this is just to notify the user, why not just DM or ping them manually; I'm not sure what this would add.

Tangentially related, I have a lot of experience with this kind of feature; the previous moderation bot that I built had a `note` feature, which effectively allowed moderators to add entries to the modlog without taking a specific action like a mute or a ban. However, this ended up being detrimental, as what would end up happening was some users would just repeatedly get warned for things - but never actually punished - because each individual offense never felt "big enough" to warrant the punishment. That's why I developed the strike system; so that you can feel confident giving out strikes for even minor offenses, and just scale your punishments to control how impactful each strike is.

TL;DR This might seem like a good feature in theory, but in practice it tends to lead to unfair or a lack of moderation. However, I am open to discussion if there is a valid use-case that I don't cover here.

**Comment by Shotty-Gunny** [comment]
> Tangentially related, I have a lot of experience with this kind of feature; the previous moderation bot that I built had a `note` feature, which effectively allowed moderators to add entries to the modlog without taking a specific action like a mute or a ban. However, this ended up being detrimental, as what would end up happening was some users would just repeatedly get warned for things - but never actually punished - because each individual offense never felt "big enough" to warrant the punishment. That's why I developed the strike system; so that you can feel confident giving out strikes for even minor offenses, and just scale your punishments to control how impactful each strike is.

I agree with your take on using the strikes, but maybe a note system on a per-user base might be useful to add a note for fellow moderators.
